relay-sampling/src/lib.rs

```diff
@@ -272,6 +273,16 @@ impl RuleCondition {
     }
 }
 
+/// Sampling rule Id
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq)]
+pub struct RuleId(pub u32);
+
+impl Display for RuleId {
```

Comment:
To match the style used in other modules, could you please import only `std::fmt` and then qualify this as `fmt::Display`. Same for `fmt::Formatter`.